sisl/physics/electron.py

```diff
@@ -735,7 +735,12 @@ def berry_phase(contour, sub=None, eigvals=False, closed=True):
     eigvals : bool, optional
        return the eigenvalues of the product of the overlap matrices
     closed : bool, optional
-       whether or not to include the connection of the last and first points in the loop (unwanted for the Zak phase)
+       whether or not to include the connection of the last and first points in the loop
+    method : string, optional
+       'berry' will return the usual integral of the Berry connection over the specified contour
+       'zak' will compute the Zak phase for 1D systems by performing a closed loop integration but
+       taking into account the Bloch factor exp(-i 2\pi/a x) accumulated over a Brillouin zone,
```

Comment:
Here you may change exp(...) to
```
:math:`e^{-i2\pi/a x}`
```
to show in LaTeX format.

Reply:
Yes, much better.